cli/cluster/cmd/uri.go

```diff
@@ -123,6 +122,17 @@ func ParseUriOpts(uri *url.URL) (UriOpts, error) {
 	return opts, nil
 }
 
+func parseUriOpts(uri *url.URL) (UriOpts, error) {
```

Comment:
Or what the idea? Why do we need a separate call?
```suggestion
func parseUriOptsWithDefaults(uri *url.URL) (UriOpts, error) {
```

Reply:
Yes, it's the idea, applied.

Reply:
I threw this patch away, so I close the conversation.